src/Map/ProspectorOverlayLayer.cs

```diff
@@ -306,19 +403,28 @@ private void OnChatMessage(int groupId, string message, EnumChatType chattype, s
             if (pos == null || groupId != GlobalConstants.InfoLogChatGroup || !_triggerwords.Any(triggerWord => message.StartsWith(triggerWord)))
                 return;
 
-            message = _cleanupRegex.Replace(message, string.Empty);
             var posX = pos.X / _chunksize;
             var posZ = pos.Z / _chunksize;
-            _messages.RemoveAll(m => m.X == posX && m.Z == posZ);
-            _messages.Add(new ProspectInfo(posX, posZ, message));
-            _clientApi.SaveDataFile(Filename, _messages);
+            _prospectInfos.RemoveAll(m => m.X == posX && m.Z == posZ);
```

Comment:
Wouldn't a Dictionary with a tuple or a position be more efficient in terms of updating (seeing as this will take longer the more entries the player collected)

Reply:
Definitely! But since that's how it's been handled so far, and a change would mean that I'd have to convert old saves to a new data format before I can open a game, I'll leave it that way for now